docs/source/using/profiles.adoc

```diff
@@ -12,7 +12,26 @@ toc::[]
 == Overview
 
 {project} allows to create multiple instances of {project}.
```

Comment:
"Minishift allows to create multiple instances of Minishift." - that does not sound right. Maybe multiple instances of the Minshift VM? Also it should be "creating" not "to create".